tools/pot/setup.py

```diff
@@ -16,17 +16,22 @@
 
 def generate_pot_version():
     try:
-        branch_name = subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]).strip().decode()
-        commit_hash = subprocess.check_output(["git", "rev-parse", "HEAD"]).strip().decode()
-        return "custom_{}_{}".format(branch_name, commit_hash)
+        pot_dir = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(os.path.realpath(__file__))),
+                                                "openvino", "tools"))
+        branch_name = subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"], cwd=pot_dir)
+        commit_hash = subprocess.check_output(["git", "rev-parse", "HEAD"], cwd=pot_dir)
+        return "custom_{}_{}".format(branch_name.strip().decode(), commit_hash.strip().decode())
     except Exception: # pylint:disable=W0703
         return "unknown version"
 
 
 def get_version():
     version = generate_pot_version()
-    if version == "unknown version":
-        version_txt = os.path.join(os.path.dirname(os.path.realpath(__file__)), "version.txt")
+    version_txt = os.path.join(os.path.dirname(os.path.realpath(__file__)), "version.txt")
+    if version != "unknown version":
```

Comment:
"unknown version" should be defined as constant or None or True/False can be used